src/pages/jobs.tsx

```diff
@@ -0,0 +1,92 @@
+import React from "react";
```

Comment:
(nit) Next.js automatically imports this for you, so there's no need to include this line